.github/workflows/testing-workflow.yml

```diff
@@ -41,4 +41,4 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-maven-
       - name: Check changes in Debezium Testing
-        run: mvn clean install -B -pl debezium-testing -am -amd -Passembly -Dcheckstyle.skip=true -Dformat.skip=true -Drevapi.skip -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
+        run: mvn clean install -B -pl debezium-assembly-descriptors,debezium-core,debezium-testing -am -amd -Passembly -Dcheckstyle.skip=true -Dformat.skip=true -Drevapi.skip -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
```

Comment:
I added both dependencies as it looked like sometimes the package for `debezium-assembly-descriptors` was missing and the build terminated. And later it looked like a cached version of `debezium-core` (without the applied changes) was used and the changes were not active and tests were failing. I had no time to dig deeper into that but maybe @jpechane have a better idea. It might be maven cache / cache-key related.

Reply:
That was exactly the problem, it has not build/published the core module (and it is hard to guess why) and deps and either were taken from Internet or from cache with result that the code was not aligned.

Reply:
Don't you think we need to change the cache key then in order to not fetch a cache that has assets for, for example,  debezium-core and use outdated code?

Reply:
Well, you are right, we can exclude `io/debezium` dir from caching. There was originally plan that there will be a single job that will build the artifacts and then the connector jobs will not do the build but testing but it seems such implementation is not necessary with selective connector building.

Reply:
Guys, can you tell me what is the problem? :))

Reply:
Sorry, the discussion is a litlebit spread https://github.com/debezium/debezium/pull/2018#discussion_r544432542 :-)